spec/app/models/concerns/good_job/advisory_lockable_spec.rb

```diff
@@ -51,7 +51,7 @@
           FROM "good_jobs"
           WHERE "good_jobs"."id" IN (
             WITH "rows" AS #{'MATERIALIZED' if model_class.supports_cte_materialization_specifiers?} (
-              SELECT "good_jobs"."id", "good_jobs"."id"
```

Comment:
Seems this expectation was wrong from the start. I don't think this would execute, same as in the issue